src/main/java/me/ryanhamshire/GriefPrevention/events/ClaimTransferEvent.java

```diff
@@ -0,0 +1,84 @@
+package me.ryanhamshire.GriefPrevention.events;
+
+import me.ryanhamshire.GriefPrevention.Claim;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import java.util.UUID;
+
+/**
+ * This Event is thrown when a claim is being transferred. If it is cancelled the claim will not be transferred.
+ * <p>
+ * Created by bertek41 on 30/10/2021.
```

Comment:
Either YYYYMMDD or MMDDYYYY

Reply:
but in general, no datestamp is really needed, and I can't be bothered to care when imma branch this off at some point anyway.